src/SourceBuild/tarball/content/repos/sdk.proj

```diff
@@ -39,8 +39,7 @@
     <RepositoryReference Include="runtime" /> -->
     <RepositoryReference Include="msbuild" />
     <RepositoryReference Include="source-build" />
-    <!-- TODO:  https://github.com/dotnet/source-build/issues/2279
-    <RepositoryReference Include="nuget-client" /> -->
+    <RepositoryReference Include="nuget-client" />
```

Comment:
Please pull in latest and make this same change in the installer project.

Reply:
Rebased and added this dependency back.